pos_config_show_accounting/models/pos_config.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Eficent <https://www.eficent.com/>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import fields, models
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    is_installed_account_accountant = fields.Boolean(
+        compute="_compute_is_installed_account_accountant")
+
+    def _compute_is_installed_account_accountant(self):
+        super(PosConfig, self)._compute_is_installed_account_accountant()
```

Comment:
Not sure if this line is usefull. (real question, not a blocking point)

Otherwise, LGTM :+1: Thanks !